components/ai_chat/renderer/ai_chat_resource_sniffer_throttle_unittest.cc

```diff
@@ -183,7 +182,7 @@ class MockDelegate : public blink::URLLoaderThrottle::Delegate {
       case MOJO_RESULT_SHOULD_WAIT:
         return 0;
       default:
-        NOTREACHED_IN_MIGRATION();
+        NOTREACHED();
     }
     return 0;
```

Comment:
Is it possible to remove the `default:` entry and just have `NOTREACHED()` outside the switch? That would help detecting new cases, if they got added.

Reply:
Force pushed an update to fix this, https://github.com/brave/brave-core/pull/26494/commits/babdb91402f60ba31f30a258259bef6919149be9